tests/from_base64-04/test.rules

```diff
@@ -0,0 +1,6 @@
+# input pcap contains a query to http://home.regit.org/?arg=dGhpc2lzYXRlc3QK
+# "dGhpc2lzYXRlc3QK" is "thisisatest\n"
+alert tcp any any -> any any (msg:"from_base64: no-decode [mode rfc4648]"; flow:to_server,established;  http.uri; content:"/?arg="; from_base64: set_error; absent;  sid:1; rev:1;)
+alert tcp any any -> any any (msg:"from_base64: no-decode with or_else [mode rfc4648]"; flow:to_server,established;  http.uri; content:"/?arg="; from_base64: set_error; absent: or_else; content: "foobar";  sid:2; rev:1;)
```

Comment:
`absent: or_else; content: "foobar";` looks strange to me, what did you mean ?
Did you mean `absent: or_else; content: !"foobar";` ?